styles.css

```
@@ -1,6 +1,40 @@
 @charset 'UTF-8';
 
-h1 {
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+nav {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0 2.4rem;
+  font-size: 1.2rem;
   font-family: Arial, Helvetica, sans-serif;
-  color: blueviolet;
+  background-color: #f3f3f3;
+}
+
+nav a {
+  text-decoration: none;
+}
+
+a:link {
+  color: black;
+}
+
+a:visited {
+  color: black;
+}
+
+img.logo {
+  width: 2.5rem;
+}
```

```suggestion
nav img.logo {
  width: 2.5rem;
}
```
Seria interessante definir esse estilo apenas para o logo dentro do elemento `nav`